shared/src/main/java/com/couchbase/lite/ReadOnlyArray.java

```diff
@@ -88,7 +88,7 @@ public Number getNumber(int index) {
      */
     @Override
     public int getInt(int index) {
-        return fleeceValue(index).asInt();
+        return (int)fleeceValue(index).asInt();
```

Comment:
Fleece Integer is equivalent with Java Long.
I implemented JNI to return long. Otherwise, Java could not support Long.MAX and Long.MIN.